monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -0,0 +1,56 @@
+# Portefueille Monero, démarrage rapide
+*22/04/2019*  
+Almutasim  
+_**Les portefeuilles impliquent l'une des décisions les plus importantes que doivent prendre les utilisateurs.**_
+
+Un portefeuille logiciel Monero est un programme permettant de stocker, envoyer et recevoir des Monero. Il vous donne le contrôle absolu sur vos Monero en toute confidentialité et sans intermédiaire. Personne ne peut connaître ou modifier l'ensemble des détails de vos transactions, à moins que vous n'ayez décidé de partager vos clefs de lecture à des fins d'audit, ce qui est l'essence même de l'expérience Monero.
+
+Pour faire l'expérience de ce pouvoir, vous devrez choisir un portefeuille. L'aspect le plus critique de tout portefeuille réside dans sa crédibilité. Seuls les portefeuilles recommandés par la communauté (et par le site [getmonero.org](https://getmonero.org/fr/)) sont mentionnés dans cet article. Ensuite, vous voudrez choisir un portefeuille compatible avec votre environnement préféré - iPhone, Android ou ordinateur. Enfin, vous devrez choisir un portefeuille qui répond à vos besoins de confidentialité. Cet article vous guidera sur ce point.
+
+Certains portefeuilles fonctionnent en utilisant leur propre copie de la chaîne de blocs de Monero. On appelle également cela faire fonctionner un nœud complet. Cela vous permet de contrôler la rapidité de vos transactions. Cependant, vous pouvez vous appuyer sur un nœud distant sans risquer perdre vos Monero, vous dépendez alors de la disponibilité du nœud distant (vous serez toujours en mesure de recevoir des Monero, mais vous pourriez ne pas pouvoir en envoyer). Un autre atout du fonctionnement de votre propre nœud complet réside dans la fierté de contribuer au réseau Monero. Les nœuds complets sincères rendent Monero plus fort. Apprenez-en plus ici : [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node).
```

Comment:
Suggestion: 'leur propre copie de la chaîne de blocs de Monero' --> 'leur propre copie du blockchain de Monero'  
Reason: This is a correct translation of the meaning, but the word 'blockchain' is now commonly used for describing the technology (although it is of english origin).

Reply:
Nope. Blockchain has been translated to "Chaîne de blocs" everywhere, starting from the wallets, to the website, and other stuffs. Let's not start doing "anglicisme" where things are well established.